src/test/resources/json/WikiTest/testWikiMapping/registrations/ng9em/wikis/index.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I would say if this file is not being used by the test, then we should remove it from this PR.  It's hard to tell if it is necessary.

I would keep the comments file, because we'll want to test that in the future.  (I think I have a PR that will help with that).